app/stores/NetworkStatusStore.js

```diff
@@ -25,9 +26,17 @@ export default class NetworkStatusStore extends Store {
   }
 
   @computed get syncPercentage(): number {
-    if (this.networkDifficulty > 0 && this._localDifficultyStartedWith != null) {
+    if (this.networkDifficulty > 0 && this._localDifficultyStartedWith !== null) {
```

Comment:
It was on purpose to check against `!= null` because that includes the check against `undefined`, otherwise `undefined` would be a valid value here!